solidity/contracts/AugurConstantProductMarketRouter.sol

```diff
@@ -117,6 +117,28 @@ contract AugurConstantProductRouter {
 		shareToken.sellCompleteSets(acpm.augurMarketAddress(), msg.sender, msg.sender, setsToSell, bytes32(0));
 	}
 
+	function closeOut(IAugurConstantProduct acpm, uint256 deadline) external {
+		require(block.timestamp < deadline, "AugurCP: Deadline");
```

Comment:
For @KillariDev, we should probably set the "deadline" provided by the UI to something very long like an hour.  These days on Ethereum it is incredibly rare to not get your transaction included quickly, and a longer deadline gives the user time to review the transaction.  I do think there is value in including the deadline just in case something weird happens so the user doesn't get a surprise trade a day or two later.